apps/next/src/app/components/nav/nav.scss

```diff
@@ -0,0 +1,64 @@
+@import '~@dynatrace/fluid-design-tokens/global/theme';
+@import '~@dynatrace/fluid-design-tokens/global/spacing';
+
+:host {
+  width: 100vw;
+  font-family: sans-serif;
+}
+
+.next-nav {
+  background-color: var(--alternative-background-color);
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+}
+
+.next-nav-wrapper {
+  display: flex;
+}
+
+.next-theme-switch {
+  padding-left: $fluid-spacing--small;
+  display: flex;
+  align-items: center;
+}
+
+.next-nav-item {
+  padding-left: $fluid-spacing--small;
+}
+
+.next-nav-list {
+  display: flex;
+  justify-content: flex-start;
+  padding-left: $fluid-spacing--small;
+  list-style-type: none;
+}
+
+.next-nav-brand {
+  font-weight: bold;
+  padding: $fluid-spacing--small;
+  font-size: large;
+  display: flex;
+  align-items: center;
+}
+
+.next-nav-link {
+  text-decoration: none;
+  padding-bottom: $fluid-spacing--2x-small;
+}
+
+.next-barista-theme {
+  display: flex;
+  align-items: center;
+  padding-right: 1rem;
+}
+
+.next-back-to-barista {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: small;
```

Comment:
NIT: Not certain that font-size is already in the tokens - if so this should also use it.